ui/components/app/confirm-page-container/index.scss

```
@@ -3,6 +3,12 @@
 @import 'confirm-detail-row/index';
 @import 'confirm-page-container-navigation/index';
 
+.confirm-page-container {
+  &__dialog {
+    margin: 0.75rem 1rem;
```

I've been pushing for using pixel units for bounding box values just to save some cognitive load from Figma to code. There is a great article by Josh Comeau on this https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/

```suggestion
    margin: 12px 16px;
```